src/wbase/Task.cc

```diff
@@ -38,14 +38,14 @@
 // Third-party headers
 #include <boost/algorithm/string/replace.hpp>
 
+#include "../http/RequestBodyJSON.h"
```

Comment:
Should be:
```
#include "http/RequestBodyJSON.h"
```

Reply:
The IDE has interesting ideas about how and where headers should be included, thanks.